canonical/ubuntu.com
# Remove silicon page and custom styles, add redirect

## Done

- Follow up of https://github.com/canonical/canonical.com/pull/984, removed /silicon and customs styles
- Added redirect to c.com silicon partner page

## QA

- View the site locally in your web browser at: https://ubuntu-com-13048.demos.haus/silicon
- See that you are properly redirected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5209
